README.md

```diff
@@ -6,44 +6,116 @@
 
 [![Netlify Status](https://api.netlify.com/api/v1/badges/38c2a5f4-bac5-4bec-8d11-a9f08b5b6f71/deploy-status)](https://app.netlify.com/sites/dreamy-keller-b8ad36/deploys)
 
-START HERE: Project Setup Guidelines
-===============
+# TABLE OF CONTENTS
 
-- We use React;
-- Know how to run tsc with babel: https://babeljs.io/blog/2018/08/27/7.0.0
-- Lets agree on JS Guideline
+1. [ Description. ](#desc)
+2. [ Guidlelines ](#guide)
+3. [ Installation ](#ins)
+4. [ Running ](#run)
+5. [ Frameworks and servivices used ](#fra)
+6. [ Contributing ](#cont)
+7. [ Credits ](#cre)
 
-Technical Setup
-===============
+<a name="desc"></a>
 
-#### Run docker image to start instance.
+## 1. Description
 
-```bash
+This is a part of project XCoV19 which aims to to build a software that guides patients to the nearest hospital, while modeling the spread of the virus in order to better anticipate hospital needs. [Here](https://www.covidsos.net/website/home.html) is the official page.
```

Comment:
https://www.covidsos.net is enough for the link, since subroutes may change in future.